### DISCUSSION TITLE
Porting to .NET MAUI

### DISCUSSION THREAD

**Discussion by criss02-cs** [kwsch/PKHeX#3965] [Ideas]
Hi all, I'm a software developer, lately I've been working with the new .NET framework called .NET MAUI. I saw that your application is written with the Windows Forms framework which is a bit outdated, how about switching to this framework? It also allows you to manage cross-platforms, so for example even Mac users could download your application. I am waiting for an answer, I am also willing to help if you need

**Comment by santacrab2** [comment]
https://github.com/santacrab2/PKHeXMAUI

Work in Progress

**Comment by kwsch** [comment]
More context here: https://github.com/kwsch/PKHeX/issues/3623

Most of my attention is focused on the backend to make features possible & legality checks as accurate/performant as possible. There's about ~100 different WinForms controls/forms that have been built, which would be unfun to port :(

People are able to run PKHeX on Linux assuming they can winebottle stuff; PKHeX doesn't use any OS-specific functionality:
https://appdb.winehq.org/objectManager.php?sClass=application&iId=20317
https://projectpokemon.org/home/forums/topic/62968-pkhex-on-linux/
